### PR TITLE
minimal alphine image added

### PR DESCRIPTION
**Pull request by ZenLiuCN**
|type|former|minimal|
|------|------|---------|
|size|>120MB(compressed)|98.8MB(none compressed)|
|groonga component|full|without documents and http components|
|dev headers|yes|no|

### CODE REVIEW

File: alpine/10-minimal/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM postgres:10-alpine
+RUN apk --no-cache add build-base lz4-dev msgpack-c-dev
+
+ENV PGROONA_VER='2.2.0' \
+    GROONGA_VER='9.0.3'
```
Comment (kou) [inline alpine/10-minimal/Dockerfile]: We can use 9.0.4 here.

File: alpine/10-minimal/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
+FROM postgres:10-alpine
+RUN apk --no-cache add build-base lz4-dev msgpack-c-dev
```
[the remark this answers is not in the document]
Comment (ZenLiuCN) [reply alpine/10-minimal/Dockerfile]: I'm not sure about LZ4 or Zstandard under alpine, maybe try that next weekend.

**Comment by kou** [comment]
You don't need to create a new pull request.
You can push more commits to this branch.

**Comment by ZenLiuCN** [comment]
maybe we could use 

> You don't need to create a new pull request.
> You can push more commits to this branch.

hadn't used like that,i will try later.
ps: is 2.2.0-alpine-slim-11 look better?

**Comment by kou** [comment]
> ps: is 2.2.0-alpine-slim-11 look better?

We need to apply the same changes as 10 to 11.

**Comment by ZenLiuCN** [comment]
> > ps: is 2.2.0-alpine-slim-11 look better?
> 
> We need to apply the same changes as 10 to 11.

of couse, I'm mean use `slim` as a  version marker

**Comment by kou** [comment]
Ah, sorry. I misunderstood.
OK. We'll use `slim` instead of `minimal`.

**Comment by ZenLiuCN** [comment]
I 'm doing testing on my computer,it'll take some time cause of network is real slow. I will update code later.

**Comment by kou** [comment]
OK.

**Comment by ZenLiuCN** [comment]
codes are updated. Pls check if I've missed something.

**Pull request merged**